core/commands/name.go

```diff
@@ -53,6 +48,7 @@ Resolve the value of a reference:
 
 	Subcommands: map[string]*cmds.Command{
 		"publish": PublishCmd,
+		"upload":  UploadNameCmd,
```

Comment:
Given that record stores will be simple stores with get/put interfaces, something like `/record/put` sounds like a good name to me